src/node.cc

```diff
@@ -2849,58 +2847,103 @@ void ProcessEmitWarning(Environment* env, const char* fmt, ...) {
   f.As<v8::Function>()->Call(process, 1, &arg);
 }
 
+static bool pullFromCache(Environment* env,
+                          const FunctionCallbackInfo<Value>& args,
+                          Local<String> module,
+                          Local<Object> cache) {
+  Local<Object> exports;
+  if (cache->Has(env->context(), module).FromJust()) {
+    exports = cache->Get(module)->ToObject(env->isolate());
```

Comment:
Can you use the overloads of `Get()` and `ToObject()` that take a `Local<Context>`?

Reply:
done, had to do a bit of checks for the maybe